src/print.c

```diff
@@ -855,37 +855,6 @@ safe_debug_print (Lisp_Object arg)
     }
 }
 
```

Comment:
At the end of the file you also need to remove the `defsubr` line which refers to this function.

Reply:
ok